client.go

```diff
@@ -21,12 +21,16 @@ const (
 	ClientKey  = "cert/client.key"
 )
 
+type Dailer func(target string) (*grpc.ClientConn, error)
```

Comment:
Minor typo: Dailer --> Dialer and requires comment if exported.

Reply:
Damn... I always mix that up. Deamon, Dialer... Thanks for noticing :smile: